e2e/adult_income/data_loader.py

```diff
@@ -2,7 +2,7 @@
 
 from tqdm import tqdm
 
-from persia.prelude import PersiaBatch
+from persia.embedding.data import PersiaBatch
```

Comment:
As we have `non_id_type_feature` in `PersiaBatch`, it seems better to import from `persia.data` or `persia.dataflow` .

Reply:
Due `non_id_type_feature` is `optional`, I think the `PersiaBatch` is not suitable to place into `persia.data`.And the major functionality of `persia.data` will focus on procssing `dataset` and `dataloder`.

Reply:
I am a little worried that users mistakenly think that we are mapping non id type features to embeddings through some method, if he didn’t read the document carefully. Maybe this worry is redundant.

Reply:
I think we need an issue to remind us to refactor this.

Reply:
Related to #185